### PR TITLE
[HIBIKE] Fix some use of data structures

### PR DESCRIPTION
**Pull request by baby-bell**
Hibike had a rigid way of looking at devices that made it difficult to remove or add devices on the fly. This PR consolidates threads and queues into a single structure.

### CODE REVIEW

File: hibike/hibike_process.py [hunk 1]
```diff
@@ -101,57 +102,57 @@ def hibike_process(bad_things_queue, state_queue, pipe_from_child):
...
 
-    for read_thread in read_threads:
-        read_thread.start()
-    for write_thread in write_threads:
-        write_thread.start()
+    for (ser, uid) in smart_sensors.items():
+        index = serial_names.index(ser)
+        serial_port = serials[index]
+        pack = namedtuple('Threadpack', ['read_thread', 'write_thread',
+                                         'write_queue'])
```
Comment (kevinji) [inline hibike/hibike_process.py]: Probably want to consistently use double quotes here.

**Comment by baby-bell** [comment]
I have tested this on real hardware, and it works. Please merge if there are no other issues to address.

**Review by brandonxxlee — APPROVED** [review]
lgtm

File: hibike/hibike_process.py [hunk 2]
```diff
@@ -15,10 +16,10 @@
 __all__ = ["hibike_process"]
 
 
-UID_TO_INDEX = {}
 # .04 milliseconds sleep is the same frequency we subscribe to devices at
 BATCH_SLEEP_TIME = .04
 
+
```
Comment (brandonxxlee) [inline hibike/hibike_process.py:22]: unnecessary newline diff

**Review by dHutchings — APPROVED** [review]
lgtm, i think.

**Pull request merged**